public/js/monitor.js

```diff
@@ -81,59 +81,70 @@ function addInvalidMessage() {
   addClass(".input-group", "invalid");
 }
 
-function enableBtnIfEmailValid(e) {
-  removeInvalidMessage();
-  const emailBtn = document.getElementById("submit-email");
-  if (isValidEmail(e.target.value)) {
-      emailBtn.disabled = false;
-      removeInvalidMessage();
-  } else {
-      emailBtn.disabled = true;
-      if(e.code === "Enter") {
-        addInvalidMessage();
-      }
-  }
-}
-
-function enableBtnIfInputEmpty(e) {
-  const emailBtn = document.getElementById("submit-email");
-  if (!document.querySelector("input[name=email]").value) {
-      emailBtn.disabled = false;
-      removeInvalidMessage();
-  } else {
-      enableBtnIfEmailValid(e);
-  }
-}
-
-function hideFalseDoor() {
-  addClass("#false-door", "hidden");
-  document.body.style.overflow = "scroll";
-}
+const removeModalThings = function() {
+  removeClass("body", "show-subscribe-modal");
+  document.querySelector("#subscribe-fxa-btn").removeEventListener("click", doOauth);
+  document.getElementById("subscribe-modal").removeEventListener("click", subscribeModalClickListener);
+  document.querySelector("#subscribe-fxa-btn").removeEventListener("click", doOauth);
+  window.removeEventListener("keydown", subscribeModalKeyListener);
+};
 
-function showFalseDoor() {
+function handleSignUpModal() {
   handleEvents("SignUp");
-  removeClass("#false-door", "hidden");
-  document.body.style.overflow = "hidden";
-  const falseDoor = document.getElementById("false-door");
-  window.addEventListener("keydown", function falseDoorKeyListener(e) {
+  addClass("body","show-subscribe-modal");
+
+  const subscribeModal = document.getElementById("subscribe-modal");
+  window.addEventListener("keydown", function subscribeModalKeyListener(e) {
     if (e.code !== "Enter" && e.code !== "Escape") {
       return;
     }
-    hideFalseDoor();
-    window.removeEventListener("keydown", falseDoorKeyListener);
+    removeModalThings();
   });
-  falseDoor.addEventListener("click", function falseDoorClickListener (e) {
-    if (e.target === falseDoor || e.target === document.getElementById("close-false-door")) {
-      hideFalseDoor();
-      falseDoor.removeEventListener("click", falseDoorClickListener);
+  subscribeModal.addEventListener("click", function subscribeModalClickListener (e) {
+    if (e.target === subscribeModal || e.target === document.getElementById("close-subscribe-modal")) {
+      removeModalThings();
     }
   });
+  document.querySelector("#subscribe-fxa-btn").addEventListener("click", ()=>{
```

Comment:
Probably a minor thing, but should we do a check that `document.querySelector()` doesnt return null/undefined before trying to add an event listener?